cloud-deployment-guide/editor-plugin-version.md

```diff
@@ -20,308 +20,117 @@ This URL specifies the latest and quality assured release of {{site.productname}
-```
-
-### Advanced Code Editor
-
-* [Developer documentation]({{site.baseurl}}/plugins/premium/advcode/)
-* [Supported versions](http://cdn.tiny.cloud/1/no-api-key/tinymce-plugins/advcode/available-versions)
-
-#### Example: Specifying the Advanced Code Editor plugin version
-
-```js
-<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?advcode=2.0.0" referrerpolicy="origin"></script>
-```
-
-### Enhanced Media Embed
-
-* [Developer documentation]({{site.baseurl}}/plugins/premium/mediaembed/)
-* [Supported versions](http://cdn.tiny.cloud/1/no-api-key/tinymce-plugins/mediaembed/available-versions)
-
-#### Example: Specifying the Enhanced Media Embed plugin version
-
-```js
-<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?mediaembed=2.0.0" referrerpolicy="origin"></script>
-```
-
-### Link Checker
-
-* [Developer documentation]({{site.baseurl}}/plugins/premium/linkchecker/)
-* [Supported versions](http://cdn.tiny.cloud/1/no-api-key/tinymce-plugins/linkchecker/available-versions)
-
-#### Example: Specifying the Link Checker plugin version
-
-```js
-<script src="https://cdn.tiny.cloud/1/no-api-key/tinymce/5/plugins.min.js?linkchecker=2.0.0" referrerpolicy="origin"></script>
-```
-
-## Specifying a self-hosted deployment of features/plugins
+## Specifying the version of premium plugins deployed from Tiny Cloud
 
-If you're deploying [only premium plugins from {{site.cloudname}}]({{ site.baseurl }}/cloud-deployment-guide/features-only), you may want to have some features served from {{site.cloudname}} and some features served from your self-hosted installation. This is also possible with URL query parameters and the special version name "SDK".
+Each TinyMCE version is bundled with a set of premium plugins, but it is possible to specify different versions of each premium plugin to use with TinyMCE. Use the URL query parameters to specify the version of each premium plugin to load. This approach works with both the [{{site.productname}} editor and premium plugins deployment via {{site.cloudname}}]({{ site.baseurl }}/cloud-deployment-guide/editor-and-features) or just the [premium plugins deployment from {{site.cloudname}}]({{ site.baseurl }}/cloud-deployment-guide/features-only).
 
-The "SDK" version lets the {{site.productname}} Plugin Manager know that you're not using {{site.cloudname}} version of a particular plugin and that it shouldn't serve the plugin from {{site.cloudname}}. It also won't display any warning or error messages if you're not entitled to it.
+The `name` of the plugin is used as a query parameter. This table summarises the possible options.
 
-### Using the Self-hosted Mentions plugin with Tiny Cloud
+| Description | Name |  Supported Versions |
+| ---- | ------------------|  ------------------ |
+| [Mentions]({{site.baseurl}}/plugins/premium/mentions/) | mentions   |  [Versions](http://cdn.tiny.cloud/1/no-api-key/tinymce-plugins/mentions/available-versions) |
```

Comment:
This was just preserving what's there. There are some plugins missing here that we will need to add.

* casechange
* export
* advtable

Probably some others...

- [ ] update plugin list